clab/options_clab.go

```diff
@@ -169,7 +173,7 @@ func WithTopoFromLab(labName string) ClabOption {
 		filter := []*types.GenericFilter{
 			{
 				FilterType: "label",
-				Field:      clabels.Containerlab,
+				Field:      containerlablabels.Containerlab,
```

Comment:
lots of this type of thing -- just the importas w/ aliases to be hella specific and explicit -- things like "labels' or "exec" or w/e could be in a bunch of packages we use -- this makes it super clear its *our* labels/exec/etc.